it/test/uk/gov/hmrc/minorentityidentification/repositories/JourneyDataRepositoryISpec.scala

```diff
@@ -18,12 +18,13 @@ package uk.gov.hmrc.minorentityidentification.repositories
 
 import play.api.libs.json.{JsString, Json}
 import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import uk.gov.hmrc.http.test.HttpClientV2Support
 import uk.gov.hmrc.minorentityidentification.assets.TestConstants.{testInternalId, testJourneyId}
 import uk.gov.hmrc.minorentityidentification.utils.{ComponentSpecHelper, JourneyDataMongoHelper}
 
 import java.util.concurrent.TimeUnit
 
-class JourneyDataRepositoryISpec extends ComponentSpecHelper with JourneyDataMongoHelper {
+class JourneyDataRepositoryISpec extends ComponentSpecHelper with JourneyDataMongoHelper with HttpClientV2Support {
```

Comment:
I don't think we need HttpClientV2 support here. This class only tests the Mongo repository.